modules/ROOT/pages/showcase-apps/index.adoc

```diff
@@ -0,0 +1,8 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Showcase Apps
+
+There are showcase apps available in all mobile supported platforms that show the provided functionality.
```

Comment:
We usually link the content here from `service-intros.adoc` but i'm not sure if it makes sense for the showcase apps.

Reply:
nah, I wasn't able to make that work :( Would look ugly.